src/components/timing/TimingControls.vue

```diff
@@ -96,23 +112,55 @@ const handleStop = async () => {
       <v-container class="w-100 pa-0 d-flex flex-column align-center">
         <v-sheet class="w-100 d-flex flex-column justify-space-around ma-0" max-width="400">
           <div v-if="!dateStorage.timeIsRunning">
-            <h3>Project</h3>
-            <v-select
-              v-if="projectsStorage.currentCustomerProjects.length >= 1"
-              label="Select Project"
-              :items="projectsStorage.currentCustomerProjects"
-              v-model="projectField"
-              :menu-icon="ArrowDown"
-              :rules="[rules.required]"
-            ></v-select>
-            <p v-else>
-              <v-btn class="ma-0 pa-0" variant="plain" color="black" :to="{ name: 'projects' }"
-                >Add a new project</v-btn
+            <h3 class="text-h5 mb-5">Project</h3>
+            <div class="d-flex">
+              <v-select
+                v-if="projectsStorage.currentCustomerProjects.length >= 1"
+                label="Select Project"
+                :items="projectsStorage.currentCustomerProjects"
+                v-model="projectField"
+                :menu-icon="ArrowDown"
+                :rules="[rules.required]"
+              ></v-select>
+
+              <!--Button appears when no projects exist-->
+              <v-btn
+                v-else
+                class="mb-5"
+                color="primary"
+                variant="outlined"
+                :to="{ name: 'projects' }"
+              >
+                Add new Project</v-btn
               >
-            </p>
+              <!--This button only appears when there are projects in existence-->
+              <v-btn
```

Comment:
Does not look like a button. Maybe the naming can be changed to "new" or just a "+", or maybe the same style than the other button but with tertiary color and "+" ¿?

Reply:
Try this:              
 <PrimaryButton
                v-if="projectsStorage.currentCustomerProjects.length != 0"
                size="x-small"
                text="+"
                class="mb-8 ml-5 mt-5"
                :to="{ name: 'projects' }"
              />
In my opinion now the style is according the style web site.

Reply:
Also when create a button, use the PrimaryButton component.

Reply:
change applied :heavy_check_mark: